### PR TITLE
Added CoinLayer API

### PR DESCRIPTION
**Pull request by lcrea**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [X] Your additions are ordered alphabetically
- [X] Your submission has a useful description
- [X] The description does not end with punctuation
- [X] Each table column should be padded with one space on either side
- [X] You have searched the repository for any relevant issues or pull requests
- [X] Any category you are creating has the minimum requirement of 3 items
- [X] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | CoinBin | Cryptocurrency information | No | Yes | Unknown | [Go!](https://coinbin.org/) |
 | CoinDesk | Bitcoin Price Index | No | No | Unknown | [Go!](http://www.coindesk.com/api/) |
 | Coinigy | Interacting with Coinigy Accounts and Exchange Directly | `apiKey` | Yes | Unknown | [Go!](https://coinigy.docs.apiary.io) |
+| CoinLayer | Real-time Crypto Currency Exchange Rates JSON Rest API | `apiKey` | Yes | Unknown | [Go!](https://coinlayer.com) |
```
Comment (davemachado) [inline README.md]: Please remove "API" from the description. Everything here is an API so it adds nothing of value :)

File: README.md [hunk 2]
```diff
@@ -173,6 +173,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | CoinBin | Cryptocurrency information | No | Yes | Unknown | [Go!](https://coinbin.org/) |
 | CoinDesk | Bitcoin Price Index | No | No | Unknown | [Go!](http://www.coindesk.com/api/) |
 | Coinigy | Interacting with Coinigy Accounts and Exchange Directly | `apiKey` | Yes | Unknown | [Go!](https://coinigy.docs.apiary.io) |
+| CoinLayer | Real-time Crypto Currency Exchange Rates JSON Rest | `apiKey` | Yes | Unknown | [Go!](https://coinlayer.com) |
```
Comment (davemachado) [inline README.md]: Also don't need "JSON Rest"

Comment (lcrea) [reply README.md]: Done! I hope it's okay now.

**Comment by davemachado** [comment]
Thanks!

**Pull request merged**